test/elixir/test/jwt_roles_claim_test.exs

```diff
@@ -0,0 +1,126 @@
+defmodule JwtRolesClaimTest do
+  use CouchTestCase
+
+  @testdb "jwttestdb"
+
+  @global_server_config [
+    %{
+      :section => "chttpd",
+      :key => "authentication_handlers",
+      :value => [
+                  "{chttpd_auth, jwt_authentication_handler}, ",
+                  "{chttpd_auth, cookie_authentication_handler}, ",
+                  "{chttpd_auth, default_authentication_handler})"
+                ] |> Enum.join
+    },
+    %{
+      :section => "jwt_keys",
+      :key => "hmac:myjwttestkey",
+      :value => ~w(
+        NTNv7j0TuYARvmNMmWXo6fKvM4o6nv/aUi9ryX38ZH+L1bkrnD1ObOQ8JAUmHCBq7
+        Iy7otZcyAagBLHVKvvYaIpmMuxmARQ97jUVG16Jkpkp1wXOPsrF9zwew6TpczyH
+        kHgX5EuLg2MeBuiT/qJACs1J0apruOOJCg/gOtkjB4c=) |> Enum.join()
+    }
+  ]
+
+  setup do
```

Comment:
Hi @big-r81 
Just a question, do you need to create a testdb? It seems that the _session endpoint is the only involved here.

Reply:
Hi, oh yes, we dont' need it! Good catch! First I wanted to evaluate `$db/_security` and found the way to the `_session` endpoint. So I can remove setup/0 and tear_down/0.